test_image_detection_fix.py

```
@@ -0,0 +1,59 @@
+        'linked_messages': []
+    }
+    
+    with patch('image_handler.create_image_data_url', new_callable=AsyncMock) as mock_create:
+        mock_create.return_value = "data:image/png;base64,test123"
+        
+        data_urls = await get_all_images_from_context(message_context)
+        
+        print(f"✓ Extracted {len(data_urls)} data URL(s)")
+        assert len(data_urls) == 1, "Should extract 1 data URL"
+        assert data_urls[0] == "data:image/png;base64,test123", "Should create data URL"
+        print(f"✓ Data URL created successfully")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)